continuous_integration/recipes/distributed/meta.yaml

```
@@ -3,79 +3,103 @@
 {% set version = (major_minor_patch[:2] + [new_patch]) | join('.') + environ.get('VERSION_SUFFIX', '') %}
 {% set dask_version = environ.get('DASK_CORE_VERSION', '0.0.0.dev') %}
 
+{% set proc_version = "1.0.0" %}
+{% set proc_build_number = "0" %}
+{% set cython_enabled = build_ext == "cython" %}
+
 
 package:
-  name: distributed
+  name: distributed-ext
```

Normally in conda-forge we have used `-split` to indicate split output

```suggestion
  name: distributed-split
```